src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -45,11 +45,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
-    <Content Include="razorLanguageService.servicehub.service.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="razorLanguageService64.servicehub.service.json">
+    <Content Include="Microsoft.VisualStudio.Razor.LanguageService.servicehub.service.json">
```

Comment:
In 16.9 the ServiceHub process is expected to always be 64-bit Core CLR process.